.github/workflows/azure-login-negative.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     - name: Run Azure PowerShell
       id: ps_1
       continue-on-error: true
-      uses: azure/powershell@v1.2.0
+      uses: azure/powershell@v1
```

Comment:
This change appears to result in the script no longer failing.

Here's a build where it failed: https://github.com/Azure/login/actions/runs/7054795167/job/19204229806

Here's the release that enabled powershell@v1 to support macOS: https://github.com/Azure/powershell/releases/tag/v1.4.0

Here's the new run w/ this change where the step succeeds:
https://github.com/Azure/login/actions/runs/7334441304/job/19971129807#step:7:46

I'm not quite sure what the goal is here.

It's trivial to write an `inlineScript` that fails:

```pwsh
exit 1
```

If that's the desired test...